presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -68,6 +71,32 @@
     @Named("databases.presto.admin_role_enabled")
     private boolean adminRoleEnabled;
 
+    // Declare these statically so they have the same IDs in every test
```

Comment:
I don't understand this comment

Reply:
I'll just remove it, it's misleading anyway.